src/main/webapp/feed.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Message Feed</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
+
+<script>
+    
+  // Fetch messages and add them to the page.
+  function fetchMessages(){
```

Comment:
Create a separate JS file for Javascript code instead of putting everything in HTML file